package.xml

```diff
@@ -42,7 +42,20 @@
         <api>stable</api>
     </stability>
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
-    <notes>${notes}</notes>
+    <notes>
+    **WARNING**: With this release the file `_generated.php` is not used anymore and files `_generated_api.php` and `_generated_internal.php` are generated instead. This only impacts users having a very custom installation procedure that requires custom builds and manual copy of files.
+
+    ### Added
+    - Bring ZAI config, internal spans, and improved exception handling to PHP 7 #1293
+    - Add properties and exceptions ZAI implementations for PHP 5 #1306
```

Comment:
This is only on master (intentionally merged after you branching off)

Reply:
It was left over in the release draft. Removed from PR description and release draft, but not here. Thanks for the good catch